template.yml

```diff
@@ -32,9 +32,23 @@ Resources:
         CodeUri: src/handler/
         Handler: post.handler
         Runtime: nodejs18.x
+        Timeout: 20
         Events:
           PostLambdaApi:
             Type: Api
             Properties:
               Path: /v3/technical-records
               Method: post
+    PatchLambdaFunction:
+      Type: 'AWS::Serverless::Function'
+      Properties:
+        CodeUri: src/handler/
+        Handler: patch.handler
+        Runtime: nodejs18.x
+        Timeout: 20
+        Events:
+          PatchLambda:
+            Type: Api
+            Properties:
+              Path: /v3/technical-records/{systemNumber}/{createdTimestamp}
```

Comment:
Is this path going to clash with any of the other patch endpoints?

Reply:
So I believe the idea is this could be potentially extended to patch other items too

Reply:
Aren't update vin, update vrm and correct the record 3 different endpoints?